src/common/base/src/mem_allocator/mmap_allocator.rs

```diff
@@ -258,15 +258,29 @@ pub mod linux {
             let mut uname = unsafe { std::mem::zeroed::<libc::utsname>() };
             assert_ne!(-1, unsafe { libc::uname(&mut uname) });
             let mut length = 0usize;
-            while length < uname.release.len() && uname.release[length] != 0 {
+
```

Comment:
How about using https://docs.rs/sys-info/latest/sys_info/fn.os_release.html and parse it by `semver`?

Reply:
I noticed that, but I do not want to have extra crate deps because here we already have fallbacks.